src/pool.h

```diff
@@ -32,9 +32,12 @@ OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 #ifndef POOL_H
 #define POOL_H
 
+#define MAX_RIG_ID 32
+
 void account_hr(double *avg, const char *address);
 uint64_t account_balance(const char *address);
 uint64_t worker_count(const char *address);
 void worker_list(char *list_start, char *list_end, const char *address);
+_Bool worker_hr(double *avg, const char *address, const char *rig_id);
```

Comment:
Use an int please. 0 for success.

Reply:
changed to int, 2 on bad/notfound data, 1 on not found account, 0 on success